source/common/config/grpc_mux_impl.h

```diff
@@ -117,6 +119,7 @@ class GrpcMuxImpl : public GrpcMux,
 
   private:
     std::list<GrpcMuxWatchImpl*>& watches_;
+    std::list<GrpcMuxWatchImpl*>::iterator it_;
```

Comment:
suggest making a `using` nickname for the list rather than typing it twice, but not a big deal.

Reply:
Thanks for the input. Is something like that `using GrpcMuxWatchImplList = std::list<GrpcMuxWatchImpl*>;` ?

Reply:
Yes though considering this is local to the class, I'd just call it `WatchList`.

Reply:
Done.